css/styles-responsive.css

```diff
@@ -80,17 +80,18 @@
   }
   .overlay {
     position: fixed;
-    bottom: 0px;
+    top: 100%;
     left: 10%;
     right: 10%;
-    display: none;
     flex-direction: column;
+    display: flex;
     width: 80%;
     padding-top: 20px;
     padding-bottom: 20px;
     background-color: #fff;
     border-radius: 13px 13px 0px 0px;
     z-index: 1;
+    transition: transform 500ms linear;
```

Comment:
Change `linear` to something like `ease-in-out`. That will look more elegant. Also isn't `500ms` a little too much?